### PR TITLE
update notebook

### PR DESCRIPTION
**Pull request by rhagarty**
- updated repo location for downloading R4ML
- updated text

### CODE REVIEW

File: notebooks/R4ML_Classification_modeling_using_SVM.ipynb [hunk 1]
```diff
@@ -473,7 +474,7 @@
    "source": [
     "##### Convert AirDelay to Categorical Variable.\n",
     "\n",
-    "In order to illustrate the classification process, next we will convert our independent variable (i.e **Y == AirDelay**) into a categorical variable.\n",
+    "In order to illustrate the classification process, we will now convert our independent variable (i.e **Y == AirDelay**) into a categorical variable.\n",
```
Comment (aloknsingh) [inline notebooks/R4ML_Classification_modeling_using_SVM.ipynb]: orginal has mistake convert (Y == AirDelay) to (Y == ArrDelay)

File: notebooks/R4ML_Classification_modeling_using_SVM.ipynb [hunk 2]
```diff
@@ -500,9 +501,8 @@
     "\n",
     "As we have learned, R4ML provides a library with built-in, scalable ML algorithms. But which algorithm should you use? You have the choice of a Support Vector Machine (SVM) or Mlogit (Multiclass logistic regression).\n",
     "\n",
-    "For this example, we will use the SVM. As the first naive approach, we will try to use the canned SVM algorithm provided by the R4ML. Before that lets talk a little about SVM.\n",
-    "\n",
-    "Before that lets talk a little about SVM\n"
+    "For this example, we will use SVM. For our initial approach, we will use the pre-canned SVM algorithm provided by R4ML. But first, that lets talk a little about SVM.\n",
```
Comment (aloknsingh) [inline notebooks/R4ML_Classification_modeling_using_SVM.ipynb]: I think having the word naive is better. what do you think?

File: notebooks/R4ML_Classification_modeling_using_SVM.ipynb [hunk 3]
```diff
@@ -987,16 +987,16 @@
    "metadata": {},
    "source": [
     "\n",
-    "Once we know this, lets evaluate our model with respect to the grid search. While the the k-fold cross-validation could be applied here, we will use the 2-fold cross-validation.\n"
+    "Once we determine the moel, lets evaluate it with respect to the grid search. While the k-fold cross-validation could be applied here, we will use the 2-fold cross-validation.\n"
```
Comment (aloknsingh) [inline notebooks/R4ML_Classification_modeling_using_SVM.ipynb]: moel should be model

**Review by aloknsingh — COMMENTED** [review]
other than those comments it lgtm

File: notebooks/R4ML_Classification_modeling_using_SVM.ipynb (hunk 2 above)
Comment (rhagarty) [reply notebooks/R4ML_Classification_modeling_using_SVM.ipynb]: I don't think we should use the word naive. But if you want it, I can put it back in.

File: notebooks/R4ML_Classification_modeling_using_SVM.ipynb (hunk 3 above)
Comment (rhagarty) [reply notebooks/R4ML_Classification_modeling_using_SVM.ipynb]: fixed

File: notebooks/R4ML_Classification_modeling_using_SVM.ipynb (hunk 1 above)
Comment (rhagarty) [reply notebooks/R4ML_Classification_modeling_using_SVM.ipynb]: fixed

**Pull request merged**